sphinx/writers/latex.py

```diff
@@ -437,7 +437,7 @@ def astext(self) -> str:
             'body': ''.join(self.body),
             'indices': self.generate_indices(),
         })
-        return self.render('latex.tex_t', self.elements)
+        return self.render('latex.tex', self.elements)
```

Comment:
Thanks for the review @AA-Turner.  On reflection I'm slightly concerned by this filename suffix adjustment to the LaTeX writer's `render` interface.

_Maybe_ it's OK as-is, but I think I could rewrite this to be more backwards compatible (in case anyone has subclassed the writer, for example, or make calls to that render method directly for some reason).